static/app/components/charts/components/tooltip.tsx

```diff
@@ -224,6 +224,8 @@ function getFormatter({
       `<div class="tooltip-date">${date}</div>`,
       `<div class="tooltip-arrow"></div>`,
     ].join('');
+
+    return `<div class="tooltip-container">${tooltipContent}</div>`;
```

Comment:
Wrapping tooltip elements (`.tooltip-series`, `.tooltip-date` and `.tooltip-arrow`) inside a container (`.tooltip-container`), so we can add a single `box-shadow` style to that container.